### PR TITLE
[MIG] Migreted pos modules from 10 to 11.

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**
- Migrated pos_backend_communication and pos_backend_partner from version 10 to 11.

### CODE REVIEW

File: pos_backend_partner/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Raphaël Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "POS Backend Customer",
+    "summary": "Choose point of sale customers in backend",
+    "version": "10.0.1.0.0",
```
Comment (Chandresh-SerpentCS) [inline pos_backend_partner/__manifest__.py]: version: 11.0.1.0.0

File: pos_backend_communication/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "POS Backend Communication",
+    "summary": "Communicate with odoo's backend from POS.",
+    "version": "11.0.1.0.0",
+    "category": "Point of Sale",
+    "website": "http://www.akretion.com",
+    'author': 'Akretion, '
+              'Serpent Consultancy Services PVT.LTD.,'
```
Comment (Chandresh-SerpentCS) [inline pos_backend_communication/__manifest__.py]: Serpent Consulting Services PVT.LTD.

File: pos_backend_partner/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    "category": "Point of sale",
+    "website": "http://www.akretion.com",
+    'author': "Akretion,Odoo, "
+              "Serpent Consultancy Services PVT.LTD., "
```
Comment (Chandresh-SerpentCS) [inline pos_backend_partner/__manifest__.py]: Change the company name as above

**Comment by JayVora-SerpentCS** [comment]
did we use format patch here? @Chandresh-SerpentCS

**Comment by Chandresh-SerpentCS** [comment]
No sir, 
All the PRs are without **format-patch** process.

**Pull request closed without merging**